example/test/amp-load.test.js

```diff
@@ -1,41 +1,6 @@
-const test = require('ava');
-
-const IFRAME_SELECTOR = '#viewer > iframe';
-
-test('AMP runtime loads in iframe', async t => {
-  const {iframe} = t.context;
-  await iframe.waitForSelector('html.i-amphtml-iframed');
-  t.pass();
-});
-
-test('AMP iframe uses CSP', async t => {
-  const {page, iframe} = t.context;
-  await page.setRequestInterception(true);
-  page.on('request', req => {
-    if (req.url().startsWith('https://evil.example/')) {
-      t.fail('CSP allowed a request to be made');
-    }
-    req.continue();
-  });
+const {loadAMP} = require('./util/loader');
 
+test('AMP runtime loads in iframe', async () => {
+  const {iframe} = await loadAMP();
   await iframe.waitForSelector('html.i-amphtml-iframed');
```

Comment:
so I am new to this but this will throw and error and fail the test if not found?

Reply:
The test has a timeout of (I think) 5s, so if this doesn't resolve, it fails due to timing out. I'll check if I can make that more explicit somehow.